### PR TITLE
Get api v1 playlists

### PR DESCRIPTION
**Pull request by andrewetobin**
__x_ Wrote Tests
__x_ Implemented
__x_ Reviewed
# Neccesary checkmarks:
- [x] All Tests are Passing

- [x] The code will run locally

## Type of change
- [x] New feature
- [] Bug Fix

# Implements/Fixes:
adds functionality for the api/v1/playlists endpoint
closes #

# Check the correct boxes

- [x] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

# Please Include a link to a gif of your feelings about this branch
Alt Text

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -66,6 +66,16 @@ app.post('/api/v1/songs', (request, response) => {
     });
 });
 
+app.get('/api/v1/playlists', (request, response) => {
```
Comment (BeccaHyland) [inline server.js]: request looks great

File: server.js [hunk 2]
```diff
@@ -68,6 +68,14 @@ app.post('/api/v1/songs', (request, response) => {
     });
 });
 
+
+app.get('/api/v1/playlists', (request, response) => {
+  database('playlists').select('playlist_name')
+    .then((songs) => {
```
Comment (KathleenYruegas) [inline server.js]: Why is this returning songs?  Shouldn't it be returning playlists?

File: test/routes.spec.js [hunk 3]
```diff
@@ -135,4 +135,17 @@ describe('API Routes', () => {
       });
     });
   });
+  describe('/api/v1/playlists', () => {
+    it("getting response from api/v1/playlists", done => {
```
Comment (BeccaHyland) [inline test/routes.spec.js]: You are also testing that playlists are returned, in addition to getting a 200 response, consider an it block description such as,  `it("returns all playlists"....) Of course, whatever is consistent with your team standard for describing test blocks is the priority.

File: server.js (hunk 2 above)
Comment (andrewetobin) [reply server.js]: yikes, sorry. good catch

**Pull request closed without merging**